hw_2.js

```diff
@@ -0,0 +1,119 @@
+"use strict"
+
+// NORMAL level
+
+// Task 1
+
+let x = 20;
+let y = 58;
+let z = 42;
+
+let value = x + y + z;
+
+console.log(value);
+
+// Task 2
+
+let secondsInMinute = 60;
+let minutesInHour = 60;
+let hoursInDay = 24;
+let daysInYear = 365;
+let age = 21;
+
+let myAgeInSeconds = age * daysInYear * hoursInDay * minutesInHour * secondsInMinute;
+
+console.log(myAgeInSeconds);
+
+// Task 3
+
+// variant1
+
+let count = 42;
+let userName = '42';
+
+count = String(count);
+userName = Number(userName);
+
+console.log(count);
+console.log(userName);
+
+console.log(typeof count);
+console.log(typeof userName);
+
+// variant2
+
+userName = +userName;
+console.log(typeof userName);
+
+
+// Task 4
+
+let a = 1;
+let b = 2;
+let c = "белых медведей";
+
+a = String(a);
+b = String(b);
+
+let bears = a + b + ' ' + c;
```

Comment:
Пропустил, так будет 3 медведя
```suggestion
let bears = `${a}${b} ${c}`;
```
или
```suggestion
let bears = a + (b + ' ' + c);
```